### PR TITLE
spinner created

### PR DESCRIPTION
**Pull request by Efekahya**
resim
https://www.figma.com/file/SBPCNjmePIwJCzqqhXF9Y8/Rick-and-Morty---Digieggs-Assessment?node-id=1069%3A6026

### CODE REVIEW

File: src/components/LoadingSpinner/styles.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.container {
```
Comment (bybash) [inline src/components/LoadingSpinner/styles.scss]: Does scss change class names after the build? If not having generic names for the classes will create awkward problems when two or more same-named classes are used on the page.

**Review by bybash — CHANGES_REQUESTED** [review]

File: src/components/LoadingSpinner/styles.scss (hunk 1 above)
Comment (Efekahya) [reply src/components/LoadingSpinner/styles.scss]: I thought that scss package handles class names like how module.css handles class names but idk if that will be the case with scss should I play safe and change class names ?

Comment (iremlaya) [reply src/components/LoadingSpinner/styles.scss]: It doesn't handle it
https://discuss.codecademy.com/t/if-we-have-multiple-css-files-with-the-same-class-name-which-ones-properties-is-applied/389104

Comment (iremlaya) [reply src/components/LoadingSpinner/styles.scss]: Make sure to give specific class names

File: src/components/LoadingSpinner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import spinner from "../../assets/spinner.png";
+import "./styles.scss";
+export default function LoadingSpinner() {
```
Comment (iremlaya) [inline src/components/LoadingSpinner/index.tsx:5]: new line

**Review by iremlaya — APPROVED** [review]

**Pull request merged**